mods/d2k/rules/defaults.yaml

```diff
@@ -267,6 +272,7 @@
 	ShakeOnDeath:
 	Sellable:
 		SellSounds: BUILD1.WAV
+	AutoTargetIgnore:
```

Comment:
But this *does* break scripted reinforcements in the missions, which can sit idly by my ConYard, chilling.
I think it would be simplest to just not change that here. Can be done in a later PR if you want.